lerobot/configs/default.yaml

```diff
@@ -57,3 +57,29 @@ wandb:
   disable_artifact: false
   project: lerobot
   notes: ""
+
+image_transform:
+  enable: true
+  # Maximum number of transforms to apply
+  max_num_transforms: 3
+  random_order: false
```

Comment:
Maybe `random_order` should default to `true`? If it defaults to `false` it might mean we think there's something special about the order written below. We don't do we?

Re: docs, maybe for `random_order: false` we need the user to understand that the order below is the one that's used, but that by changing it here, they achieve nothing. That's because the order is set in code.

Reply:
Fixed